docs/operator-manual/high_availability.md

```diff
@@ -60,7 +60,7 @@ number of allowed concurrent kubectl fork/execs.
 
 * The controller uses Kubernetes watch APIs to maintain lightweight Kubernetes cluster cache. This allows to avoid querying Kubernetes during app reconciliation and significantly improve
 performance. For performance reasons controller monitors and caches only preferred the version of a resource. During reconciliation, the controller might have to convert cached resource from
-preferred version into a version of the resource stored in Git. If `kubectl convert` fails because conversion is not supported than controller fallback to Kubernetes API query which slows down
+preferred version into a version of the resource stored in Git. If `kubectl convert` fails because conversion is not supported than controller fall back to Kubernetes API query which slows down
```

Comment:
this is incorrect. `fallback` is the correct noun to use here - the same is true for some other lines where `fallback` was replaced with `fall back`.

Reply:
I was reading it as trying to use it as a verb.

I think the correct fix is actually:
```suggestion
preferred version into a version of the resource stored in Git. If `kubectl convert` fails because conversion is not supported then controller falls back to Kubernetes API query which slows down
```

Here's the entire thing in shorter bits (`[...]`s indicating all the corrections to make it read coherently) for easier reading:

> The controller uses Kubernetes watch APIs to maintain [a] lightweight Kubernetes cluster cache.
> This [enables avoiding] querying Kubernetes during app reconciliation and significantly improve[s] performance.
> For performance reasons [the] controller monitors and caches only [the preferred] version of a resource.
> During reconciliation,
> the controller might have to convert cached resource[s] from preferred version into a version of the resource stored in Git.
> If `kubectl convert` fails because conversion is not supported [then] [the] controller falls back to Kubernetes API query which slows down reconciliation.
> In this case advice user-preferred resource version in Git.

Reply:
https://github.com/argoproj/argo-cd/compare/7fb85c4f228ef0dc34a59a9305e9fe4fbe593fcd..9c70f7566db214c206c153d61d0727848dfc0658